src/__tests__/ProductView/QueryInterfaceAttributeList.js

```diff
@@ -1,18 +1,27 @@
 import React from 'react';
 import QueryInterfaceAttributeList from '../../ProductView/TabBar/QueryInterface/QueryInterfaceAttributeList/QueryInterfaceAttributeList.js';
 import renderer from 'react-test-renderer';
-let instance;
 
-test('On change eventtype name callback test', () => {
-    const callbackMock = jest.fn();
-    renderer.create(
+let instance, component;
+const callbackMock = jest.fn();
+
+beforeEach(() => {
+    component = renderer.create(
         <QueryInterfaceAttributeList
             ref={(child) => {instance = child}}
             eventTypeAttributes={[]}
             onChangeEventTypeName={callbackMock}/>
     );
+});
+
+test('On change eventtype name callback test', () => {
     const event = {target: {value: 'test'}};
     instance.onChangeEventTypeName(event);
     expect(callbackMock).toBeCalledWith('test');
 
+});
+
+test('Form validation for query name', () => {3
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Reformat the code to have only one statement per line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://bpt-lab.org/sonarqube/coding_rules#rule_key=javascript%3AOneStatementPerLine)